openstudiocore/src/osversion/VersionTranslator.cpp

```diff
@@ -445,6 +446,42 @@ void VersionTranslator::initializeMap(std::istream& is) {
   IddFileAndFactoryWrapper iddFile = getIddFile(currentVersion);
   if (iddFile.iddFileType() == IddFileType::UserCustom) {
     oIdfFile = IdfFile::load(is,iddFile.iddFile());
+    if( currentVersion == VersionString(1,9,0) ) {
+      if( oIdfFile ) {
+        auto sizingObjects = oIdfFile->getObjectsByType(iddFile.getObject("OS:Sizing:Zone").get());
+
+        // Figure out if the OS:Sizing:Zone object looks like it is from CBECC
+        // it will have extra fields that would were not in the "real" 1.9.0 IDD
+        bool fromCBECC = false;
+        for( auto const & object : sizingObjects ) {
+          if( auto value = object.getString(2) ) {
+            if( istringEqual("SupplyAirTemperature", value.get()) ||
```

Comment:
it is certainly a heuristic. I'll check but I think it is a required field and therefore not blank.